l10n_it_sepa_bonifici/wizard/export_sepa_cbi_estero.py

```diff
@@ -0,0 +1,383 @@
+                    'line.bank_id.bank.bic', {'line': line}, gen_args)
+                # Add info for Cross Border payment
+                partner_creditor = line.partner_id
+                creditor_node = credit_transfer_transaction_info_2_27\
+                    .xpath('//Cdtr')[transactions_count_1_6-1]
+                creditor_address_node = etree.SubElement(creditor_node,
+                                                         'PstlAdr')
+                creditor_address_country_node = etree.SubElement(
+                    creditor_address_node, 'Ctry')
+                iso_country = False
+                if line.bank_id.state == 'iban':
+                    iso_country = line.bank_id.iban[:2]
+                elif partner_creditor.country_id:
+                    iso_country = partner_creditor.country_id.code
+                if not iso_country:
+                    raise UserError(
+                        _("Missing Country for Partner '%s' (payment "
+                            "order line reference '%s')") %
+                        (line.partner_id.name, line.name))
+                creditor_address_country_node.text = iso_country
+                creditor_address_line_node = etree.SubElement(
+                    creditor_address_node, 'AdrLine')
+                if partner_creditor:
+                    address = '%s %s %s' % (
+                        partner_creditor.street or '',
+                        partner_creditor.city or '',
+                        partner_creditor.country_id and
+                        partner_creditor.country_id.name or '',)
+                creditor_address_line_node.text = address[:70]
+
+                self.generate_remittance_info_block(
+                    credit_transfer_transaction_info_2_27, line, gen_args)
+
+            if pain_flavor in pain_03_to_05:
+                nb_of_transactions_2_4.text = str(transactions_count_2_4)
+                control_sum_2_5.text = '%.2f' % amount_control_sum_2_5
+        if pain_flavor in pain_03_to_05:
+            nb_of_transactions_1_6.text = str(transactions_count_1_6)
+            control_sum_1_7.text = '%.2f' % amount_control_sum_1_7
+        else:
+            nb_of_transactions_1_6.text = str(transactions_count_1_6)
+            control_sum_1_7.text = '%.2f' % amount_control_sum_1_7
+
+        # CBI required
+        # >> v8
+        # Remove the duplicate node  LclInstrm in payment
+        # CtrlSum_node = xml_root.xpath('//PmtInf//LclInstrm')[0] >CBI required
+        # CtrlSum_node.getparent().remove(CtrlSum_node) You can
+        # remove node only from parent
+        # Remove the duplicate node  CtrlSum in payment
+        # CtrlSum_node = xml_root.xpath('//PmtInf//SeqTp')[0] >> CBI required
+        # CtrlSum_node.getparent().remove(CtrlSum_node) >> You can remove
+        # node only from parent
+        # >>>>>>>>>
+        # print(etree.tostring(xml_root, pretty_print=True))
```

Comment:
I want to left the print for the future debug.
I removed the other comment lines